### PR TITLE
JSUI-3057 Modified the  "Select" and "Exclude" strings for the "aria-label" attribut of facets

### PR DESCRIPTION
**Pull request by jlblais**
Also modified the unitTest related to the changes. 
Removed tests that are not relevant anymore.

https://coveord.atlassian.net/browse/JSUI-3057

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts
Comment (ThibodeauJF) [inline unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts:168]: I would remove this test, since it's repetitive with the other one now

File: unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts
Comment (ThibodeauJF) [inline unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts:149]: i would move this test outside of the `'when the value has the state "idle"` condition since there is no difference with it being idle or selected

**Review by ThibodeauJF — APPROVED** [review]
👌 Good job!

File: unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts
Comment (jlblais) [reply unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts:168]: good point. Removed it.

File: unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts
Comment (jlblais) [reply unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts:149]: Agreed. Moved it outside of the 'described' scope.

**Comment by olamothe** [comment]
There was additional discussion on the ticket in JIRA ( JSUI-3057 ) by Benjamin. 

I think we should take them into account before merging this.

**Review by btaillon — DISMISSED** [review]
Unfortunately, I misidentified this issue. As JF pointed out, we should probably just change the labels (so only change `strings.json`.

The labels could perhaps become something like the following (I think that's what JF suggested)?
"Select ABC with 1000 results" -> "Filter by ABC with 1000 results"
"Exclude ABC with 1000 results" -> "Filter out ABC with 1000 results"

Waiting for a response from @fbeaudoincoveo.

File: strings/strings.json
Comment (fbeaudoincoveo) [inline strings/strings.json:7436]: How about:

- Select {0} with {1} -> Inclusion filter on {0}; {1} results
- Exclude {0} with {1} -> Exclusion filter on {0}; {1} results
- Sélectionner {0} avec {1} -> Filtre d'inclusion sur {0}; {1} résultats
- Exclure {0} avec {1} -> Filtre d'exclusion sur {0}; {1} résultats

**Review by fbeaudoincoveo — APPROVED** [review]

**Comment by ThibodeauJF** [comment]
You can update the title of the PR 😄

**Comment by ThibodeauJF** [comment]
Related to this, we should modify the labels on the Breadcrumbs to
```
Remove inclusion filter on {0};
Remove exclusion filter on {0};
```
You can either add them on your PR, or make a new one for it after merging this! :)

**Review by ThibodeauJF — APPROVED** [review]

**Review by btaillon — APPROVED** [review]

**Pull request merged**
